python/whylogs/core/preprocessing.py

```diff
@@ -212,6 +223,10 @@ def _process_scalar_value(value: Any) -> "PreprocessedColumn":
             float_list.append(value)
         elif isinstance(value, str):
             string_list.append(value)
+        elif isinstance(value, list) and PreprocessedColumn._is_tensorable(value):
+            tensor_list.append(value)
```

Comment:
tenderize it here?

Reply:
I'm up for doing that, feel free to tackle or I can tackle tonight. We probably want to make a NumpyView.tensors object then to put these in.